web_client/views/imageViewerWidget/geojs.js

```diff
@@ -95,7 +97,16 @@ var GeojsImageViewerWidget = ImageViewerWidget.extend({
             this._layers[annotation.id] = layer;
         }
         window.geo.createFileReader('jsonReader', {layer})
-            .read(geojson, () => this.viewer.draw());
+            .read(geojson, (features) => {
+                _.each(features || [], (feature) => {
+                    feature.selectionAPI(this._hoverEvents);
+
+                    feature.geoOn(geo.event.feature.mouseover, (evt) => this._onMouseOverFeature(evt));
+                    feature.geoOn(geo.event.feature.mouseout, (evt) => this._onMouseOutFeature(evt));
+                    feature.geoOn(geo.event.feature.mouseclick, (evt) => this._onMouseClickFeature(evt));
```

Comment:
Would it be better to capture `geo.event.feature.mouseon` and `geo.event.feature.mouseoff`, which get fired when the topmost hovered element changes?  We could capture all five `mouse*` events, as they could all be useful at some point.

Further, since the functions all do the same thing except for the event name, perhaps there should only be one function like `(evt) => this._onMouseFeature('g:mouseOverAnnotation', evt)`, where the function would trigger the specified event.